sdk/keyvault/tests.yml

```diff
@@ -16,5 +16,49 @@ jobs:
       AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
       AZURE_CLIENT_ID: $(net-keyvault-azure-client-id)
       AZURE_CLIENT_SECRET: $(net-keyvault-azure-client-secret)
-      AZURE_KEYVAULT_URL: $(net-keyvault-azure-keyvault-url)
+      AZURE_KEYVAULT_URL: $[coalesce('$(AZURE_KEYVAULT_NAME)', '$(net-keyvault-azure-keyvault-url)')]
       AZURE_KEYVAULT_TEST_MODE: Live
+
+    PreSteps:
+    - template: ../../eng/pipelines/templates/steps/azure-cli-login.yml
+      parameters:
+        ClientID: $(net-keyvault-azure-client-id)
+        ClientSecret: $(net-keyvault-azure-client-secret)
+        TenantID: $(aad-azure-sdk-test-tenant-id)
+
+    - pwsh: |
+        Write-Host "##vso[task.setvariable variable=AZURE_KEYVAULT_NAME;isOutput=true]${Env:AZURE_KEYVAULT_NAME}"
+      displayName: Generate resource name
+      name: keyvaultvars
+      env:
+        AZURE_KEYVAULT_NAME: $[format('adpnetkv{0:yyyyMMddss}', pipeline.startTime)]
+
+    - script: |
+        az group create --name "$(AZURE_KEYVAULT_NAME)rg" --location westus --tags service-directory=keyvault test-resource=true
```

Comment:
I thought we have argeed on some kind of delete-after tag that would be used for cleaning up the run-away resources.

Reply:
We should, yes, but no reason not to try to clean up right away. Could safe a few bucks.